src/pages/AuthPage.tsx

```diff
@@ -22,38 +32,60 @@ export function AuthPage() {
 
   return (
     <Card color='transparent' shadow={false}>
-      <Typography className="text-[#030391] font-bold text-3xl leading-normal lg:mt-10">
+      <Typography className='text-[#030391] font-bold text-3xl leading-normal lg:mt-10'>
         Welcome!
       </Typography>
-      <Typography className="text-[#1488D8] text-base font-normal">
+      <Typography className='text-[#1488D8] text-base font-normal'>
         Student Smart Printing Service (SSPS)
       </Typography>
       <form className='mt-20 mb-2 w-72 md:w-80 max-w-screen-lg' onSubmit={handleSubmit(submit)}>
         <div className='mb-16 flex flex-col gap-6'>
           <Input
```

Comment:
In simple form, you can put fields right in form, but in more complicated form, putting fields in separate components will make it easier to handle re-render